pkgs/development/python-modules/pabot/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, robotframework
+}:
+
+buildPythonPackage rec {
+  pname = "pabot";
+  version = "1.8.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "robotframework-pabot";
+    sha256 = "dbae4340159875026540f6456922a5893431701cbbc2d165b6b24661e2ca665c";
+    extension = "tar.gz";
```

Comment:
default
```suggestion
```

Reply:
Sorry, this file should not even have gotten into tho PR. I'll remove it and force-push.